helm/charts/keto/templates/deployment.yaml

```diff
@@ -52,6 +52,12 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.deployment.extraInitContainers}}
```

Comment:
Please follow the pattern at the top of the file using ternary operators 😉

Reply:
Done.